src/simulation/m_mpi_proxy.fpp

```diff
@@ -2333,6 +2333,13 @@ contains
 
     end subroutine s_mpi_sendrecv_capilary_variables_buffers
 
+    subroutine s_mpi_send_random_number(phi_rn)
+        real(kind(0d0)), dimension(1:100) :: phi_rn
```

Comment:
this dimension `1:100` is concerning and it should probably be `dimension(num_broadband_freq)` where `num_broadband_freq` is defined in `m_constants`.

Reply:
shoudln't this be `intent(inout)` and have a named dimension size (non magic number)

Reply:
Thanks...I am easily caught by magic numbers :/.